docs/manual/creating-a-checker.tex

```diff
@@ -1892,6 +1892,7 @@
   \<-Acfgviz=org.checkerframework.dataflow.cfg.DOTCFGVisualizer,outdir=\emph{somedir}>.
   % TODO: create the directory if it doesn't exist.
   The directory must already exist.
+  The CFG is generated and output, but no dataflow analysis is performed.
```

Comment:
`javac -processor myProcessor -Aflowdotdir=. MyClass.java` will do the dataflow analysis. So I think we should remove this line.